deploy/csv-templates/crds/rook/ceph.rook.io_cephbucketnotifications.yaml

```diff
@@ -19,17 +17,28 @@ spec:
   - name: v1
     schema:
       openAPIV3Schema:
+        description: CephBucketNotification represents a Bucket Notifications

```

Comment:
this, is coming from rook and for generating csv we remove the description in rook

Reply:
If we remove it, why is it showing up here?

Reply:
right, https://github.com/rook/rook/pull/11724 this will fix this issue.

note: this issue will faced in upstream ocs-operator and not in downstream as there we directly use `make csv` and current problem was only in `make build`

Reply:
see with https://github.com/red-hat-storage/ocs-operator/pull/1915/files#diff-f6f13ef4e3c83b5e22c436df5b1a4b24fd2c22a80d25855668b41828bcc05e12R53 this changes it is working, need to removing my tag once, included in rook release